src/libraries/Microsoft.PowerFx.Core/Types/Enums/EnumStore.cs

```diff
@@ -12,19 +12,19 @@ namespace Microsoft.PowerFx.Core.Types.Enums
     /// <summary>
     /// Static class used to store built in Power Fx enums
     /// </summary>
-    internal static class EnumStore
+    public class EnumStore
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Next step - How would hosts customize this in a reasonable way?

A lot of these look like control enums that don't belong outside of Canvas apps. 

In some cases, adding a function may want to pull in enums (ie, Date functions that consume DateTimeFormatEnumString enum may want to pull that in).

Reply:
You can customize the enums by overriding the EnumDict property in a derived class.

Reply:
can you add a test case that demonstrates the customization? Just to show a e2e.